content/en/monitors/create/types/error_tracking.md

```diff
@@ -0,0 +1,126 @@
+
+## Create an Error Tracking monitor
+
+To create an Error Tracking monitor in Datadog, navigate to [**Monitors** > **New Monitor** > **Error Tracking**][4].
+
+<div class="alert alert-info"><strong>Note</strong>: There is a default limit of 1,000 Error Tracking monitors per account. <a href="/help/">Contact Support</a> to lift this limit for your account.</div>
+
+### Select the alerting condition
+
+Alerts are grouped by issue ID and are based on error volume.
+
+Choose **Count** to alert on issues with a high number of errors and **New Issue** to alert on issues that occur for the first time.
+
+### Define the search query
+
+{{< tabs >}}
+{{% tab "Count" %}}
+
+For **Web and Mobile Apps**, select **Count**. As you expand your search filter, the Count graph above **Select the alerting condition** updates.
+
+1. Optionally, select a query suggestion such as **Error Occurrences**, **Impacted Users**, and **Impacted Sessions** to alert on. 
+2. Select **Web and Mobile Apps** or **Backend Services** and choose to monitor over a count, facet, or measure.
+   - For error occurrences, monitor over an overall count based on the issue ID.
+   - For impacted users, monitor over a unique count of user emails based on the issue ID or over a measure. 
+   - For impacted sessions, monitor over a unique count of session IDs based on the issue ID. 
+   - Monitor over a measure. If you select a measure, the monitor alerts over the numerical value of the RUM facet (similar to a metric monitor). Select an aggregation type (`min`, `avg`, `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`). 
+   
+   There are three quickfilter options to access the most frequently used facets:
+
+   - **Error Occurrences**: Triggers when the error count is `above` or `above or equal to`.
+   - **Impacted Users**: Triggers when the number of impacted user emails is `above` or `above or equal to`.
+   - **Impacted Sessions**: Triggers when the number of impacted session IDs is `above` or `above or equal to`.
+
+4. Construct a search query using the same logic as a [RUM Explorer search][1] for the issues’ error occurrences.
+5. Configure the alerting grouping strategy (optional).
+   - Simple-Alert: Simple alerts aggregate over all reporting sources. You receive one alert when the aggregated value meets the set conditions. If the query has a `group by` and you select **Simple-Alert**, you get one alert when one or multiple groups’ values breach the threshold. You may use this strategy to reduce notification noise.
+   - Multi-Alert: Multiple alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you can group a query by `@browser.name` to receive a separate alert for each browser when the number of errors is high.
```

Comment:
This is already documented in /monitors/create/configuration/#alert-grouping. Let's just point there instead.
```suggestion
5. Optionally, configure the alerting grouping strategy. For more information, see [Monitor Configuration]
```
and point to /monitors/create/configuration/#alert-grouping